src/Models/Link.php

```diff
@@ -243,9 +296,13 @@ public function getURL(): string
      * Get all link types except the generic one
      *
      * @throws ReflectionException
+     * @deprecated 3.0.0 Will be removed without equivalent functionality to replace it
      */
     private function getLinkTypes(): array
     {
+        Deprecation::withNoReplacement(function () {
+            Deprecation::notice('3.0.0', 'Will be removed without equivalent functionality to replace it.');
```

Comment:
```suggestion
     * @deprecated 3.0.0 Use SilverStripe\LinkField\Services\LinkTypeService::generateAllLinkTypes() instead.
     */
    private function getLinkTypes(): array
    {
        Deprecation::withNoReplacement(function () {
            Deprecation::notice('3.0.0', 'Use SilverStripe\LinkField\Services\LinkTypeService::generateAllLinkTypes() instead.');
```
Again, exact wording so long as it matches the others - main point is this has a replacement in v4

Reply:
Updated